guides/contributor/CLA.md

```diff
@@ -0,0 +1,88 @@
+<!-- cspell:ignore Docu heptio gitdm -->
```

Comment:
Such exceptions are usually handled via https://github.com/open-telemetry/community/blob/main/.cspell.yaml in this repo.